### ISSUE TITLE
Is it possible to support multiple tabs combined vertical and horizontal splits

### ISSUE THREAD

**Issue by wenlzhang** [gitobsidiantutorial/obsidian-tabs#11]
Is it possible to support opening multiple tabs on more panes, e.g. 2 by 2 panes? I did not figure out how to make this work.

**Comment by gitobsidiantutorial** [comment]
Do mean things like [Visual Studio grid tabs](https://code.visualstudio.com/assets/updates/1_25/grid-layout.gif)? and splitting a tab horizontally?

That isn't supported as of now, although it is possible to have multiple tabs. This is limited to one type of split, with no nesting.

[![video](https://raw.githubusercontent.com/gitobsidiantutorial/obsidian-tabs/main/vids/horizontalconfigure.webp)](https://raw.githubusercontent.com/gitobsidiantutorial/obsidian-tabs/main/vids/horizontalconfigure.webp)

**Comment by wenlzhang** [comment]
Yes. Exactly like the attached Visual Studio grid tabs/panes, where one can switch among multiple tabs in each one of the 2 by 2 grid.

When organizing notes during the GTD Weekly Review, I often need to open four (or more) notes 2 by 2 at the same time such that I can refer to other files when summarizing notes in one file. So, it would be really great if this can be supported in the future.

Maybe this is too much to request, but it would be even more convenient if one can drag a file for the file navigation, starred file list, or the recent files list [tgrosinger/recent-files-obsidian: Display a list of most recently opened files](https://github.com/tgrosinger/recent-files-obsidian) to each of the split/tab.
